src/generator/generator.controller.js

```diff
@@ -188,6 +195,11 @@ function addPostRunCheckIn (plays) {
     return [...plays, new SpecialPlay('special:post-run-check-in', hosts, templates.special.postRunCheckIn)];
 }
 
+function addComplianceReportPlay (plays) {
+    const hosts = _(plays).flatMap('hosts').uniq().sort().value();
```

Comment:
Retrieval of hosts is from the existing play here is a bit odd and done several times. This might need some refactoring ideas. What are your thoughts?

Reply:
Remediations in general seems really bad about making redundant calls, to inventory in particular, but I'd prefer to address that in a separate task.  Basically we need to review each endpoint and clean things up systematically.